src/App.tsx

```diff
@@ -14,11 +17,113 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Props = {};
+type State = {
+  isStarted: boolean,
+  goodsList: string[],
+  visibleGoods: string[],
+  isReversed: boolean,
+};
+
+class App extends React.Component<Props, State> {
```

Comment:
```suggestion
type State = {
  isStarted: boolean,
  goodsList: string[],
  visibleGoods: string[],
  isReversed: boolean,
};

class App extends React.Component<{}, State> {
```

Reply:
If `Props` has no actual purpose, there is no need to create one. Its generic is replaceable with `{}`